server/auth/auth_test.go

```diff
@@ -1560,15 +1567,17 @@ func TestMatchStatus(t *testing.T) {
 	}
 
 	reqPayload[0].MatchID = []byte{}
-	err = rig.mgr.handleMatchStatus(user.conn, req)
-	if err == nil {
```

Comment:
This was essentially `var err error = *msgjson.Error`, so `err` could never be nil regardless of the pointer value.

Reply:
Oh, I see you resolved this independently in the txData pr @buck54321.  I'm still changing the return variable to `msgErr` to highlight that the return type is a concrete type (`*msgjson.Error`) and not just an `error`.